jobs/quoscient_cybersecurity_germany_permanent.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Senior Python Developer
+company: QuoScient GmbH
+url: http://quoscient.io
+location: Frankfurt am Main, Germany
+contract: permanent
+contact:
+    name: Ina Klemm
+    email: careers@quoscient.io
+created: !!timestamp '2017-03-01' 
+tags:
+  - germany
+  - python 3
+  - cyber security
+  - NoSQL
+  - senior
+  - OOP
+  - Git
+  - Unix
+---
+
+# Overview
+
+QuoScient is a specialist in Information Security Services. We are providing security integration support, incident response consultancy and a distributed analysis platform that is supporting collaboration and information sharing.  
```

Comment:
Grammar check: "We are **provide** security integration support, incident response consultancy and a distributed analysis platform that **supports** collaboration and information sharing."